lib/kiev/logger.rb

```diff
@@ -54,8 +56,11 @@ class Logger
         entry.merge!(data)
       elsif !data.nil?
         entry[:message] = data.to_s
+        entry[:status] = ERROR_STATUS if data.to_s.downcase.include?(LOG_ERROR)
```

Comment:
I tested with different scenarios and it comes in the else statement only in case of an error. I haven't found any success case that was in the else statement.